### PR TITLE
Logging no method or interval

### PR DESCRIPTION
**Pull request by pawelcodilime**

### CODE REVIEW

File: plugin.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins:
     psutil:
         executor: host_agent
-        source: https://github.com/cloudify-cosmo/cloudify-psutil-plugin/archive/1.0.zip
+        source: https://github.com/cloudify-cosmo/cloudify-psutil-plugin/archive/logging-no-method-or-interval.zip
```
Comment (ran-z) [inline plugin.yaml]: Not sure why you need this here for this specific pull request..?

File: psutil_agent/loop.py [hunk 2]
```diff
@@ -115,6 +115,14 @@ def scheduled_fun():
 
             scheduled_fun()
 
+        if 'method' not in config:
```
Comment (ran-z) [inline psutil_agent/loop.py:118]: (maybe we should have a couple of basic unit tests for these?)

File: plugin.yaml (hunk 1 above)
Comment (pawelcodilime) [inline plugin.yaml]: Of course it shouldn't be here. My mistake.

File: psutil_agent/loop.py (hunk 2 above)
Comment (pawelcodilime) [inline psutil_agent/loop.py:118]: Yes, it would probably make some sense.

**Pull request merged**